src/main/flight/mixer.c

```diff
@@ -779,29 +780,24 @@ void mixTable(void)
         int16_t rollPitchYawMixRange = rollPitchYawMixMax - rollPitchYawMixMin;
         int16_t throttleRange, throttle;
         int16_t throttleMin, throttleMax;
+        static int16_t throttlePrevious = 0;   // Store the last throttle direction for deadband transitions in 3D
```

Comment:
fair enough.

I reversed the logic there and changed initialisation to mid_rc.

The arming is allowed in both directions, but starts from positive when stick was centered. Anyway it depends how the stick was moving prior arming.